BDArmory/Ammo/PooledBullet.cs

```diff
@@ -1156,11 +1156,13 @@ bool BulletHitAnalysis(RaycastHit hit, float period)
                         
 
                         // We cap the minimum L/D to be 1.2 to avoid that edge case in the pen formula
-                        if (massRatio < 1.2f * caliber / length)
+                        if ((massRatio * (length-10f) + 10f) < (1.1f * caliber))
                         {
-                            bulletMass = 1.2f * bulletMass * caliber / length;
+                            float ratio = (1.1f * caliber - 10f) / (length - 10f);
```

Comment:
I think something has to be wrong with the approach here, since this adjustment can give negative ratios for small calibre bullets.

Reply:
Hmmm, that's right, there's a boundary condition for caliber = 10. The issue here is that our minimum projectile length is 10  mm due to legacy code. I can fix that piece of legacy code, however that would cause everyone's calculated apBulletMod values to have to change. I could also just set that limit there and use the original formula instead for those, I think I'll do the latter.